 RosRate: 40
 leg_num: 4
 motor_num: 12
 sensor_num: 23
 
 controlParam: 
     - MI
     - PCPGBeta
     - CPGType
     - PSN_L1
     - VRN_hip_L1
     - VRN_knee_L1
     - MNB1_L1
     - MNB2_L1
     - MNB3_L1
     - PSN_L2
     - VRN_hip_L2
     - VRN_knee_L2
     - MNB1_L2
     - MNB2_L2
     - MNB3_L2
     - PSN_L3
     - VRN_hip_L3
     - VRN_knee_L3
     - MNB1_L3
     - MNB2_L3
     - MNB3_L3
     - PSN_L4
     - VRN_hip_L4
     - VRN_knee_L4
     - MNB1_L4
     - MNB2_L4
     - MNB3_L4

 MI: 0.15
 PCPGBeta: 0.75
 CPGType: 5

 PSN_L1: 0.0
 VRN_hip_L1: 0.04
 VRN_knee_L1: 0.04

 PSN_L2: 0.0
 VRN_hip_L2: 0.04
 VRN_knee_L2: 0.04

 PSN_L3: 0.0 
 VRN_hip_L3: 0.04
 VRN_knee_L3: 0.04

 PSN_L4: 0.0 
 VRN_hip_L4: 0.04
 VRN_knee_L4: 0.04

 MNB1_L1: 0.0
 MNB2_L1: -0.00
 MNB3_L1: -0.00

 MNB1_L2: 0.0
 MNB2_L2: -0.00
 MNB3_L2: -0.00

 MNB1_L3: 0.0
 MNB2_L3: -0.00
 MNB3_L3: -0.00

 MNB1_L4: 0.0 
 MNB2_L4: -0.00
 MNB3_L4: -0.00


 subscribeTopic:
   - sensorValue
   - terminateController
 advertiseTopic:
   - motorValue
   - neuroNetworkOutput
   

